src/icons/Icon.js

```diff
@@ -46,11 +48,17 @@ const Icon = props => {
             alignItems: 'center',
             justifyContent: 'center',
           },
-        ]}
+          disabled && styles.disabled,
+          disabled && disabledStyle,
+        ])}
+        {...onPress && { disabled }}
```

Comment:
just to improve readability should should we put the conditional statement in parens ? ` {...(onPress && { disabled })}`

Reply:
Its an object so we have to spread it. What this is doing is only adding disabled to the props if onPress is passed in.

Reply:
yeah we do spread. I am just saying we wrap that expression in parens to enhance readability 😉

Reply:
Oh lol, prettier removes unnecessary brackets.